include/grpc/impl/codegen/byte_buffer_reader.h

```diff
@@ -28,8 +28,8 @@ struct grpc_byte_buffer;
 struct grpc_byte_buffer_reader {
   struct grpc_byte_buffer *buffer_in;
   struct grpc_byte_buffer *buffer_out;
-  /** Different current objects correspond to different types of byte buffers */
-  union {
+  /** Different current objects correspond to different types of byte buffers */ 
```

Comment:
Nit: Please remove trailing whitespace.  (I suspect the clang-format sanity check will fail because of this.)